### PR TITLE
Added Network NG Modules initialization guide

### PR DESCRIPTION
**Pull request by ghost**
**ng-installation.adoc**
"Zimbra Network NG Modules: First Steps" guide

### CODE REVIEW

File: ng-installation.adoc [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+[[admin-ng]]
+Switching to Admin NG
+---------------------
+Switching to the new Admin NG module is a simple process that will migrate any
+relevant ACL information to the module's own configuration manager, clearing
+existing ACLs and ACEs from the system.
+
+The Admin NG module is significantly different than the old Delegated Administration engine.
+Please make sure to read the product's documentation and to only migrate to Admin NG if its featureset
```
Comment (plobbes) [inline ng-installation.adoc]: nit: change featureset to feature set

**Comment by jorgedlcruz** [comment]
@gordyt Can we have this on a separate public link for easy review as you did with the Admin Guide? Thank you!

**Comment by gordyt** [comment]
@zextras-contrib It doesn't look like the `ng-installation.adoc` file is actually wired into anything.  It should be properly `included` in one-or-more of the documents that are generated from this repo.  Not sure which is appropriate but there is a multi-server, single-server, and upgrade doc all generated out of this repo.  The `README` contains instructions.

Please be sure to actually generate the HTML output from the appropriate doc(s) to make sure there are not any `asciidoctor` warnings and that the resulting output is appropriate.

**Review by gordyt — CHANGES_REQUESTED** [review]
Please see my previous comment.

**Review by gordyt — APPROVED** [review]

**Comment by gordyt** [comment]
OK, structurally this looks good.  There are updates to both the multi-server and single-server guides.  I've exported them to the `gh-pages` branch and published them for your preview [here](https://zimbra.github.io/installguides/).  Just follow the links to the `8.8.0` docs.

@jorgedlcruz please take a look at the exported single-server and multi-server exports.  If they look OK, please be sure and mark this pull request as approved via the review button (upper right).

**Pull request merged**